jaeger-core/src/test/java/io/jaegertracing/internal/reporters/RemoteReporterTest.java

```diff
@@ -368,6 +368,31 @@ public int append(JaegerSpan span) {
     assertEquals(0, metricsFactory.getCounter("jaeger_tracer_reporter_spans", "result=dropped"));
   }
 
+  @Test
+  public void testUpdateErrorMetricWhenCommandExecuteFails() throws Exception {
+
+    int reporterFailures = 5;
+
+    sender = new InMemorySender() {
+      @Override
+      public int append(JaegerSpan span) throws SenderException {
```

Comment:
Can be a runtime exception used to avoid the signature change?

Reply:
so the [codepath](https://github.com/jaegertracing/jaeger-client-java/blob/5e81d89303d3a670e09664413a52fdba23783cbf/jaeger-core/src/main/java/io/jaegertracing/internal/reporters/RemoteReporter.java#L105) that we want to cover and called out in #380 has a catch on `SenderException`

Reply:
+1 just posting sender interface

```
public interface Sender {
  int append(JaegerSpan span) throws SenderException;

  int flush() throws SenderException;

  int close() throws SenderException;
}

```